### PR TITLE
Hifi: Story Page Take Action Section

### PR DESCRIPTION
**Pull request by jean-zhang**

### CODE REVIEW

File: components/about-page/main-resource/MainResource.js [hunk 1]
```diff
@@ -8,7 +8,6 @@ export default function MainResource({ mainResourceData }) {
     <OvalButton
       buttonText={getString(resourceTitle).toUpperCase()}
       href={resourceLink.url}
-      onClick={() => window.open(resourceLink.url, '_blank')}
```
Comment (isabellberzsenyi) [inline components/about-page/main-resource/MainResource.js:11]: lol nice fix

File: components/story-page/take-action/TakeAction.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  ShareLinkImage,
+  TakeActionHeadingDesktop,
+  TabletMobileWrapper,
+  TabletMobileTake,
+  TabletMobileAction,
+  TakeActionWrapper,
+  UnderlineImage,
+} from './TakeAction.styles';
+
+const copyToClipboard = () => {
```
Comment (isabellberzsenyi) [inline components/story-page/take-action/TakeAction.js:27]: beautiful

File: components/story-page/take-action/TakeAction.styles.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    align-self: center;
+    margin-right: 0;
+  }
+`;
+
+export const ShareLinkText = styled(H4)`
+  font-weight: ${fontWeights.normal};
+`;
+
+export const ShareLinkImage = styled.img``;
```
Comment (isabellberzsenyi) [inline components/story-page/take-action/TakeAction.styles.js]: nit: is this necessary or can we just use `img`

Comment (jean-zhang) [reply components/story-page/take-action/TakeAction.styles.js]: missed that -- will do

**Review by isabellberzsenyi — APPROVED** [review]
work of art

**Review by jzhzhang66 — COMMENTED** [review]
Looks really good! Only thing is the top part is cut off on my computer-
image

**Comment by jean-zhang** [comment]
@jzhzhang66 Updated to be relative to the bottom instead, let me know how it looks on your screen!

File: components/story-page/take-action/TakeAction.js [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+    <ShareLinkWrapper
+      onMouseEnter={() => setShareHover(true)}
+      onMouseLeave={() => setShareHover(false)}
+      onClick={copyToClipboard}
+      url={icons.SHARE_STORY_LINE}
+    >
+      <ShareLinkText>SHARE THIS STORY</ShareLinkText>
+      <img
+        src={shareHover ? icons.SHARE_STORY_HOVER : icons.SHARE_STORY}
+        alt="link icon"
```
Comment (josephmannis) [inline components/story-page/take-action/TakeAction.js]: Could this just be share icon instead of link icon?

**Review by josephmannis — CHANGES_REQUESTED** [review]
Looks great – noticed some bugs on Safari. The buttons are bit smaller than they should be and the animation for scrolling back doesn't seem to work. OK with going through if you get design approval from Jay!

File: components/story-page/take-action/TakeAction.js (hunk 4 above)
Comment (jean-zhang) [reply components/story-page/take-action/TakeAction.js]: Updated, also was able to find a fix for Safari scrolling!

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
If the next font weight is too bold could we make the photographer name have some shadow so it appears bolder like in the hi-fis? *especially on mobile I think the font could be thicker
image
image

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
could we remove this left to right sliding? Nvmd i think thats from the image being too big which I think you fix in the other PR
image
-----
Reply: This is a result of the image gallery, will be fixed in that PR

**Review by jzhzhang66 — COMMENTED** [review]
Can we make this whole thing bigger (especially photographer / camera) and reduce the left padding a bit like on the hi-fis? 
image

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
Reduce the back to top button to footer distance on mobile and tablet
image
image

**Pull request merged**